dali/tensorflow/daliop.cc

```diff
@@ -68,7 +68,9 @@ REGISTER_OP("Dali")
   .Attr("shapes: list(shape) >= 1")
   .Attr("num_threads: int = -1")
   .Attr("device_id: int = -1")
-  .Attr("prefetch_queue_depth: int = 2")
+  .Attr("exec_separated: bool = false")
+  .Attr("gpu_prefetch_queue_depth: int = 2")
+  .Attr("cpu_prefetch_queue_depth: int = 2")
```

Comment:
If the `exec_separated` is `false`, why `XXX_prefetch_queue_depth` is specific (i.e. 2)?

Reply:
Here is `exec_separated` is `false` we actually look at `gpu_prefetch_queue_depth`